File: Phan_A/bai34/bai_34.py
import sys

def dao_nguoc(n: int) -> int:
    sign = -1 if n < 0 else 1
    temp = abs(n)
    rev = 0
    while temp > 0:
        rev = rev * 10 + temp % 10
        temp //= 10
    return rev * sign

if __name__ == "__main__":
    input_data = sys.stdin.read().split()
    if input_data:
        n = int(input_data[0])
        print(dao_nguoc(n))

File: Phan_A/bai34/bai_34.jl
function dao_nguoc(n::Int128)::Int128
    sign = n < 0 ? -1 : 1
    temp = abs(n)
    rev = Int128(0)
    while temp > 0
        rev = rev * 10 + temp % 10
        temp = div(temp, 10)
    end
    return rev * sign
end

function main()
    input_data = split(read(stdin, String))
    if isempty(input_data)
        return
    end
    n = parse(Int128, input_data[1])
    println(dao_nguoc(n))
end

main()